services/azure/Compute/virtualMachines.yml

```diff
@@ -0,0 +1,92 @@
+name: virtualMachines
+description: >-
+  Azure Virtual Machines (Microsoft.Compute/virtualMachines) provide general-purpose compute.
+  Compromise or misuse can expose data, interrupt services, or incur spend.
+scope: CRITICAL
+notes: >-
+  Privileges below are grouped conceptually and do not map 1:1 with ARM operation names.
+  Assess in combination with disk, networking, identity, and key vault privileges.
+privileges:
+  list:
+    risks: [discovery:infra]
+    notes: >-
+      Enumerates VMs and metadata such as OS, sizes, and attached resources.
+  read:
+    risks: [discovery:infra, exfiltration:metadata]
+    notes: >-
+      Reads VM configuration and instance metadata which can reveal sensitive details.
+  createOrUpdate:
+    risks: [destruction:infra, impact:spend]
+    notes: >-
+      Can change images, identities, startup scripts, or sizes; can scale up costs.
+  delete:
+    risks: [destruction:data, destruction:infra]
+    notes: >-
+      Permanently removes a VM; attached ephemeral data may be lost.
+  start:
+    risks: [impact:spend]
+    notes: >-
+      Starting instances increases spend; may enable exposed services.
+  restart:
+    risks: [impact:dos]
+    notes: >-
+      Restarts can disrupt workloads and availability.
+  powerOff:
+    risks: [impact:dos]
+    notes: >-
+      Stops the VM without releasing compute allocation; interrupts services.
+  deallocate:
+    risks: [impact:dos]
+    notes: >-
+      Stops and releases allocation; VMs become unavailable until started.
+  runCommand:
+    risks: [exfiltration:data, impact:hijack]
+    notes: >-
+      Executes arbitrary commands inside the guest OS, enabling data theft or tampering.
+  login:
```

Comment:
There is an `initialAccess` category which seems to me like it would apply to logins.  I don't see us using that anywhere else though.

https://attack.mitre.org/tactics/TA0001/